book/4 - MultiWorlds/3 - Linking Worlds with a Plugin.html

```diff
@@ -0,0 +1,3 @@
+<p>
+	The easiest way to link worlds together is using a dedicated plugin, such as <a href="http://forum.mc-server.org/showthread.php?tid=2157">Portal V2</a>. More plugins can be found at the <a href="http://forum.mc-server.org/forumdisplay.php?fid=2">respository</a>.
```

Comment:
I believe the easiest way is to use the build-in portal system described in the file below. It's a one-time set up. The most configurable is probably using the plugin, but it requires people to manually set up the destination of the portal.